index.html

```diff
@@ -12376,6 +12608,14 @@ <h2>Relationship to RDF</h2>
       and <code>false</code>. The JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]]
       defines the <a data-cite="JSON-LD11-API#data-round-tripping">conversion rules</a>
       between JSON's native data types and RDF's counterparts to allow round-tripping.</li>
+    <li class="changed">As an extension to the <a data-cite="RDF11-CONCEPTS#data-model">RDF data model</a>,
+      <em>typed literals</em>, where the datatype is `xsd:string`, or <a>language-tagged strings</a>
+      MAY include a <a>base direction</a>.
+      As there is currently no standardized mechanism for representing the <a>base direction</a>
+      of <a>RDF literals</a>, the JSON-LD to RDF transformation loses the base direction.
+      The <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">Deserialize JSON-LD to RDF Algorithm</a>
+      also provides a means of representing <a>base direction</a>
+      using non-standardized mechanisms which will preserve round-tripping through RDF.</li>
```

Comment:
Is the goal to represent `direction` in a non-standard way when this flag is used? I would think that is much better than inventing some kind of translation to an RDF pattern that could map back in a different way. (I understand this is an ongoing discussion, just putting this out there).

Edit: I see below what the current options are. Hmm, well, apply suggestion or ignore it. Given that text I would recommend ignoring it ... but it seems like if we adopt the approach below we'll be stuck with it forever. I'd rather the community not take on that technical debt and instead output non-standard RDF with what we believe is the right way to model this stuff -- and see if it gets uptake or not. That's for another issue/discussion though.

```suggestion
      using mechanisms which will preserve round-tripping through non-standard RDF.</li>
```